deploy/kubernetes/base/controller/csidriver_info.yaml

```diff
@@ -1,5 +1,4 @@
-#### TODO: This need to be updated to v1 once we official support 1.18+
-apiVersion: storage.k8s.io/v1beta1
```

Comment:
We say that dev supports 1.17+, so we can't move to v1 csi driver yet.

Reply:
sure let me revert it.

Reply:
Done.